### PR TITLE
Lucide icon imports and usage in components

### LINKED ISSUE #2042: Replace the documentation site icon with `@yamada-ui/lucide`
### Subject

All pages

### Description

Currently, We are using the icon from `@yamada-ui/fontawesome`, but I would like to replace it with the icon from `@yamada-ui/lucide`.

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #2042

## Description

Lucide icon imports and usage in components.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by 108yen — CHANGES_REQUESTED** [review]

File: docs/contents/components/forms/autocomplete/index.ja.mdx
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/autocomplete/index.ja.mdx:1]: Fixed.

File: docs/contents/components/forms/multi-autocomplete/index.ja.mdx
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-autocomplete/index.ja.mdx:1]: Fixed.

File: docs/contents/components/forms/multi-select/index.ja.mdx
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-select/index.ja.mdx:1]: Fixed.

File: docs/contents/components/forms/native-select/index.ja.mdx
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/native-select/index.ja.mdx:1]: Fixed.

File: docs/contents/components/forms/select/index.ja.mdx
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/select/index.ja.mdx:1]: Fixed.

**Review by 108yen — APPROVED** [review]

**Pull request merged**